ui/modal/modalPublishPreview/view.jsx

```diff
@@ -151,38 +152,48 @@ const ModalPublishPreview = (props: Props) => {
   const txFee = previewResponse ? previewResponse['total_fee'] : null;
   //   $FlowFixMe add outputs[0] etc to PublishResponse type
   const isOptimizeAvail = filePath && filePath !== '' && isVid && ffmpegStatus.available;
-  let modalTitle;
-  if (isStillEditing) {
-    if (livestream) {
-      modalTitle = __('Confirm Update');
-    } else {
-      modalTitle = __('Confirm Edit');
-    }
-  } else if (livestream) {
-    modalTitle = releasesInFuture ? __('Schedule Livestream') : __('Create Livestream');
-  } else {
-    modalTitle = __('Confirm Upload');
-  }
 
+  let modalTitle;
   let confirmBtnText;
-  if (!publishing) {
+  updateLabels();
```

Comment:
This is usually not needed because effects will always run at least once after the first mount.  Currently, `updateLabels` will run twice.

But not a big deal overall since `updateLabels` is lightweight.